web-ui/components/Assets/intents.tsx

```diff
@@ -91,6 +93,65 @@ const StakingIntent: React.FC<StakingIntentProps> = ({ address, isWalletConnecte
     );
   }
 
+  if (currentNetwork.chainName === 'dydx') {
+    return (
+      <Box w="full" color="white" borderRadius="lg" p={4} gap={6}>
+        <VStack spacing={4} align="stretch">
+          <Flex gap={6} justifyContent="space-between" alignItems="center">
+            <Heading fontSize="lg" fontWeight="bold" textTransform="uppercase">
+              Stake Intent
+            </Heading>
+
+            <SignalIntentModal
+              refetch={refetch}
+              selectedOption={currentNetwork}
+              isOpen={isSignalIntentModalOpen}
+              onClose={closeSignalIntentModal}
+            />
+          </Flex>
+
+          <Flex borderBottom="1px" borderBottomColor="complimentary.900" alignItems="center" justifyContent="space-between">
+            <IconButton
+              variant="ghost"
+              _hover={{ bgColor: 'transparent', color: 'complimentary.900' }}
+              _active={{
+                transform: 'scale(0.75)',
+                color: 'complimentary.800',
+              }}
+              color="GrayText"
+              aria-label="Previous chain"
+              icon={<ChevronLeftIcon w={'25px'} h={'25px'} />}
+              onClick={handleLeftArrowClick}
+            />
+            <SlideFade in={true} key={currentChainIndex}>
+              <Text fontSize="lg" fontWeight="semibold">
+                {chains[currentChainIndex]}
+              </Text>
+            </SlideFade>
+            <IconButton
+              _active={{
+                transform: 'scale(0.75)',
+                color: 'complimentary.800',
+              }}
+              color="GrayText"
+              _hover={{ bgColor: 'transparent', color: 'complimentary.900' }}
+              variant="ghost"
+              aria-label="Next chain"
+              icon={<ChevronRightIcon w={'25px'} h={'25px'} />}
+              onClick={handleRightArrowClick}
+            />
+          </Flex>
+
+          <VStack pb={4} overflowY="auto" className="custom-scrollbar" gap={4} spacing={2} align="stretch" maxH="250px">
+            <Center mt={6}>
+              <Text fontSize="xl">Coming Soon!</Text>
+            </Center>
+          </VStack>
+        </VStack>
+      </Box>
+    );
+  }
```

Comment:
This functionality is ready and this code rabbit comment does not identify any actual issue. We are using Coming Soon! because dydx implementation is ready and just waiting for governance.